05-classification.Rmd

```diff
@@ -437,10 +442,10 @@ show the calculation of distance for each of the 5 closest points.
 -----------------     ----------------- ----------------- 
 
 
-**More than two explanatory variables**
+**More than two predictor variables**
```

Comment:
Let's make this a header instead of bold - that way we (and others) can link to it when we want to share it with others.
```suggestion
#### More than two predictor variables**
```

Reply:
(same with other instances of this in the chapter, e.g., **Summary** below)

Reply:
doesn't make sense to have a 4th level numbered heading as a subsection of a 2nd level. I also disagree with using numbered subsection headings for this sort of thing -- these aren't notionally different subsections, the boldface i'm using akin to a \paragraph{blah} command in latex. The way it typesets separates the text far too much.

On the other hand, I can imagine when you pdflatex this stuff to generate a PDF copy, those bold things don't look great.

Is there an unnumbered 3rd level subsection heading in Rmd? that would be the ideal solution

I will make a separate issue for this because this sort of thing will have to be made consistent throughout the book anyway.

Reply:
Issue #164 is open now